### ISSUE TITLE
Lua GetTeams() function

### ISSUE THREAD

**Issue by R00Ki3** [fortressforever/fortressforever#186]
Could we add a GetTeams function that would return the active teams in an array?

**Comment by squeek502** [comment]
Even better would be to allow Lua to do this directly if it's not already possible and then add the GetTeams function to includes/util/utils.lua. Functions needed for that as far as I can tell:
- [x] team:GetPlayerLimit()

What do you mean by active teams? Would those include unassigned/spectator?

**Comment by R00Ki3** [comment]
Yes you could include unassigned and spec and basically any team that doesn't have SetPlayerLimit set to -1. Yeah, team:GetPlayerLimit() Would work the same and I could make a function to use it however i want.
